### ISSUE TITLE
iOS Server doesn't work

### ISSUE THREAD

**Issue by mikosaure** [SoftwareGuy/Ignorance#51]
The server doesn't work on iOS (no matter the version) with Unity 2018.3.11f1 and XCode 10.2.1.

If I run the Mirror example `Basic`, click `LAN Server Only` then try to connect from another device, nothing happens.
It works fine when I do the exact same thing with an Android device or with Unity Editor.

**Comment by SoftwareGuy** [comment]
This might be something to do with listening addresses on iOS (had this issue before on OS X Editor and Standalone). Do you have "Bind to all interfaces" ticked?

**Comment by SoftwareGuy** [comment]
Just tried this on an iPhone 4S connected via WiFi and a Unity Editor instance on my Macbook Pro, but that looks like it was just a fluke because it doesn't want to connect again...

**Comment by SoftwareGuy** [comment]
Some weirdness was happening but I was able to get 5 out of 5 attempts of having the iPhone 4S as the host and the editor connecting to it in a row. Then Xcode started saying it randomly timed out waiting for the app to start which seemed to be cured after I unplugged the phone and replugged it.

Then I tried again on the iPhone as the host and the Editor seemed to not want to connect, so I minimized the app, stopped the Editor trying to connect, then resumed the app and pointed the Editor to the phone... boom, it connected.

Apparently something in the Xcode Editor I spotted said that iOS kills any open sockets when applications are put into background mode. But I am not sure about that.

Seems to be **random**.
